index.js

```diff
@@ -1,125 +1,201 @@
 'use strict'
 
 const nodePath = require('path')
-
-const commonExtname = require('common-extname')
-const commonPathPrefix = require('common-path-prefix')
-const glob = require('glob')
+const GlobSync = require('glob/sync').GlobSync
```

Comment:
The documentation also says:
> sync DEPRECATED: use glob.sync(pattern, opts) instead.

Which means if I apply the same logic to the class, I need to use the `GlobSync` class as well. It does appear that the `GlobSync` class is exported in `require('glob')` as well, so I can change that at least.